### PR TITLE
Restructure Webapp UI

### PR DESCRIPTION
**Pull request by Maxr1998**
- Move CoordinatorLayout to WebViewFragment, since it's only supposed to be used there anyways
- Fade in WebView once the app has connected to the sever
- Fix a potential NPE when setting the locale

The background icon as discussed in #436 is still missing, but this should be easy to add now. I will play with that tomorrow.

cc: @CarlosOlivo

### CODE REVIEW

File: app/src/main/java/org/jellyfin/mobile/utils/UIExtensions.kt [hunk 1]
```diff
@@ -46,6 +48,18 @@ fun View.applyWindowInsetsAsMargins() {
...
+fun View.fadeIn() {
+    alpha = 0f
+    isVisible = true
+    with(animate()) {
+        alpha(1f)
+        @Suppress("MagicNumber")
+        duration = 300L
+        interpolator = LinearOutSlowInInterpolator()
+        start()
+    }
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/mobile/utils/UIExtensions.kt:60]:
```suggestion
    animate().apply {
        alpha(1f)
        @Suppress("MagicNumber")
        duration = 300L
        interpolator = LinearOutSlowInInterpolator()
        withLayer()
    }
```
- start() is not needed, will automatically start
- withLayer() enables hardware acceleration

File: app/src/main/res/layout/fragment_webview.xml [hunk 2]
```diff
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<WebView xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (nielsvanvelzen) [inline app/src/main/res/layout/fragment_webview.xml:2]: could probably use a framelayout here

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/jellyfin/mobile/utils/UIExtensions.kt (hunk 1 above)
Comment (Maxr1998) [reply app/src/main/java/org/jellyfin/mobile/utils/UIExtensions.kt:60]: - neat, didn't know
- that's not the default??

File: app/src/main/res/layout/fragment_webview.xml (hunk 2 above)
Comment (Maxr1998) [reply app/src/main/res/layout/fragment_webview.xml:2]: Nope, we need that for the snackbar.

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**